src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,100 @@
+<script>
+export default {
+  props: {
+    location: {
+      type: Object,
+      required: true,
+      validator: val => val.type && val.id,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    append: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  inject: ['girderRest'],
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  asyncComputed: {
+    breadcrumb: {
+      default: { root: {}, path: [] },
+      async get() {
+        this.loading = true;
+        const breadcrumb = { root: {}, path: [] };
+        const { id, type, name } = this.location;
+        if (type === 'folder') {
+          // The last breadcrumb isn't returned by rootpath.
+          if (name) {
+            breadcrumb.path.unshift(this.location);
+          } else {
+            const { data } = await this.girderRest.get(`folder/${id}`);
+            breadcrumb.path.unshift(this.extractCrumbData(data));
+          }
+          // Get the rest of the path.
+          const { data } = await this.girderRest.get(`folder/${id}/rootpath`);
+          data.reverse().forEach((crumb) => {
+            const { object } = crumb;
+            const entity = this.extractCrumbData(object);
+            if (entity.type === 'folder') {
+              breadcrumb.path.unshift(entity);
+            } else {
+              breadcrumb.root = entity;
+            }
+          });
+        } else {
+          const { data } = await this.girderRest.get(`${type}/${id}`);
+          breadcrumb.root = this.extractCrumbData(data);
+        }
+        this.loading = false;
+        return breadcrumb;
+      },
+    },
+  },
+  methods: {
+    extractCrumbData(object) {
+      return {
+        name: object._modelType !== 'user' ? object.name : object.login,
+        id: object._id,
+        type: object._modelType,
+        parentId: object.parentId,
+        parentType: object.parentCollection,
+      };
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+v-breadcrumbs.girder-breadcrumb-component.font-weight-bold.pa-0
+  v-icon.mdi-24px(
+      :disabled="disabled",
+      slot="divider",
+      color="accent") {{ $vuetify.icons.chevron }}
```

Comment:
The "accent" color makes these dividers look like links, which they aren't. The default is that they are grey, which I think I like better. @jtomeck are you OK to change the color?

Reply:
@zachmullen I don't know that it's super important for me to push back on because it really just comes down to opinion, so if you really think it's needed I'm fine with changing the color. I'll give my opinion on it anyway though in case anyone's interested :P

From a designer's perspective it seems like many people are of the mindset that giving something a color makes it look like a link. I think it all depends on the usage. For example, if I had a block of copy and changed the color of a few words, sure they would look like a link. I don't think things like giving a prominent headline a color, or giving dividers a color make them look like links, especially since nothing happens on hover and the cursor does not change.